dashpay/src/main/kotlin/org/dashevo/dashpay/Profiles.kt

```diff
@@ -52,16 +53,35 @@ class Profiles(
         )
     }
 
-    fun get(userId: String) : Document? {
+    fun get(userId: String): Document? {
         val query = DocumentQuery.Builder()
             .where("\$userId", "==", userId)
             .build()
         try {
             val documents = platform.documents.get(typeLocator, query)
             return if (documents.isNotEmpty()) documents[0] else null
-        } catch (e : Exception) {
+        } catch (e: Exception) {
             throw e
         }
     }
 
+    fun getList(
+        userIds: List<String>,
+        retrieveAll: Boolean = true,
+        startAt: Int = 0
+    ): List<Document> {
+        val documentQuery = DocumentQuery.Builder()
+        documentQuery.whereIn("\$userId", userIds)
```

Comment:
I forgot what we could search for multiple profiles in one query.  On some of these parameters there is a limit of 100 items that can be included.

Reply:
There may be an exception thrown if userIds contains more than 100 items.

Reply:
When running a query, I get this error message in the trailers of the `StatusRuntimeException (INVALID_ARGUMENT, Invalid Query)`

```javascript
Metadata(server=nginx/1.17.9,date=Wed, 03 Jun 2020 15:22:54 GMT,content-type=application/grpc,content-length=0,0={"name":"JsonSchemaValidationError","keyword":"enum","dataPath":".where[0][1]","schemaPath":"#/properties/where/items/oneOf/0/items/1/enum","params":{"allowedValues":["<","<=","==",">",">="]}},1={"name":"JsonSchemaValidationError","keyword":"const","dataPath":".where[0][1]","schemaPath":"#/properties/where/items/oneOf/1/items/1/const","params":{"allowedValue":"in"}},2={"name":"JsonSchemaValidationError","keyword":"const","dataPath":".where[0][1]","schemaPath":"#/properties/where/items/oneOf/2/items/1/const","params":{"allowedValue":"startsWith"}},3={"name":"JsonSchemaValidationError","keyword":"const","dataPath":".where[0][1]","schemaPath":"#/properties/where/items/oneOf/3/items/1/const","params":{"allowedValue":"elementMatch"}},4={"name":"JsonSchemaValidationError","keyword":"const","dataPath":".where[0][1]","schemaPath":"#/properties/where/items/oneOf/4/items/1/const","params":{"allowedValue":"length"}},5={"name":"JsonSchemaValidationError","keyword":"const","dataPath":".where[0][1]","schemaPath":"#/properties/where/items/oneOf/5/items/1/const","params":{"allowedValue":"contains"}},6={"name":"JsonSchemaValidationError","keyword":"oneOf","dataPath":".where[0]","schemaPath":"#/properties/where/items/oneOf","params":{"passingSchemas":null}})
```

It is saying that the only allowed values are `"<","<=","==",">",">="` for this query, which doesn't include IN.

Reply:
This issue was resolved in dapi-client-android: https://github.com/dashevo/dapi-client-android/pull/5